src/controllers/login.js

```diff
@@ -0,0 +1,10 @@
+const pageTitle = require('./../views/helpers/makeTitle');
+
+exports.get = (request, response) => {
+  response.render('login', {
+    js: ['collectData', 'login'],
+    css: ['login'],
+    layout: 'login_signup',
+    title: pageTitle.titleName('login'),
```

Comment:
@AhmedAl-Almi I think you need a slight change:
`title: pageTitle('login')`

.. because you set module.exports directly, in makeTitle.js

Reply:
In the first was as you say but we change on file "makeTitle"
#76

Reply:
we used exports as object and function inside this object

Reply:
Ah ok!

Reply:
@AhmedAl-Almi  as Tom says you need to change it, by using it in helpers